addons/purchase/models/purchase.py

```diff
@@ -178,6 +186,11 @@ def name_get(self):
             result.append((po.id, name))
         return result
 
+    @api.onchange('date_planned')
+    def onchange_date_planned(self):
+        if self.date_planned:
+            self.order_line.date_planned = self.date_planned
```

Comment:
shouldn't date_planned becomes a compute/inverse depending on the date planned of the lines? similar to expected_dates

Reply:
I discussed with oma. She doesn't want the date on the PO to be changed.